packages/ferium-bin/ferium-bin.pacscript

```diff
@@ -17,4 +18,19 @@ build() {
 
 install() {
   sudo install -Dm755 ferium "${STOWDIR}/${name}/usr/bin/${pkgname}"
+  mkdir -p ~/.config/ferium
+  # create a default config or else ferium will fail to do completions
+  echo '{
```

Comment:
Tell the user that a default config has been installed in their home directory, as you're modifying their home directory